Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/PrefabToInMemorySpawnableNotificationBus.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzFramework/Entity/EntityContext.h>
+
+namespace AzToolsFramework::Prefab
+{
+    //! This EBus is used to send notifications during the conversion of a prefab template to an in-memory spawnable
+    class PrefabToInMemorySpawnableNotifications
+        : public AZ::EBusTraits
```

Comment:
This bus is not protected by a recursive mutex like the previous one - are there potential concerns with threading here?

Reply:
That's a good callout but the recursive mutex should no longer be necessary as this replacement notification is smaller in scope and should only be called on the main thread.